packages/components/src/card/types.ts

```diff
@@ -9,7 +9,7 @@ import type { CSSProperties } from 'react';
  */
 import type { Props as SurfaceProps } from '../surface/types';
 
-export type SizeOptions = 'xSmall' | 'small' | 'medium' | 'large';
+export type SizeOptions = 'xSmall' | 'small' | 'medium' | 'large' | 'none';
```

Comment:
I'd like to point out that `size: 'none'` feels a bit counter-intuitive. What does it even mean? With how `none` behaves in CSS and UIs often, does it mean it'll be invisible? It feels confusing and I think we should either reconsider the naming (maybe use `minimal` or if @sarayourfriend has a better naming idea), or document it better if we really need to have a `none`.

Reply:
That's a fair point, when I proposed `none` as a new value I didn't consider the confusion it may create around its meaning in CSS.

The reason to add `none` here was to have an option for `<Card />` users to remove completely the internal padding.
Renaming it to `minimal` definitely sounds good to me!

Reply:
> IMHO the confusion comes from the fact that we're working with padding, but we're expressing it as `size`

1000%. Unfortunately, this prop already existed in the previous implementation of `<Card />`, and I didn't want to introduce a breaking change.

I will go ahead and rename it `minimal`

Reply:
FWIW @ciampo and I spent a great deal of time tossing around ways to rename the size prop to something more sensible and nothing made sense.

We _could_ soft-deprecate `size` and rename it to `padding`... What do you think Marco? Is it worth it? Then the `none` would make sense.

`minimal` doesn't make sense to me because it's not `minimal` size. Maybe we should just allow it to be falsy and in that case it renders the `none` case 🤔 Bah, these naming things are so hard!

Reply:
Naming things is indeed hard! In the CS literature it's often compared to be as hard as cache invalidation 😅 

I don't have a strong feeling to use `minimal` but `none` definitely feels confusing as I described above.

Reply:
Should I remove the `none` / `minimal` change from this PR?

We can then take our time and discuss the best approach in a separate PR (ie. coming up with a good alternative to `none`, or soft-deprecate it in favour of some better name for the prop itself)

Reply:
Sounds good to me to do it in another PR 👍

Reply:
Reverted the change in https://github.com/WordPress/gutenberg/pull/32566/commits/14373263491b7cd003e2d8c8c9f88e41972d22ac. This change will be discussed in a separate PR